8.x/getting-started/documentation.md

```diff
@@ -2,4 +2,466 @@
 
 [[toc]]
 
-Larammerce is getting documented, as well as developed at high speed. So if you want to be on track, follow the guides.
\ No newline at end of file
+This section guides you to write a document for larammerce project .
```

Comment:
this article is a complete guide to writing documentation for the larammarce project.